### PR TITLE
build: avatar component

### PR DESCRIPTION
**Pull request by Shubhamkr2610**

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
+@import url("/index.css");
+@import url("/components/sidebar.css");
```
Comment (purohitdheeraj) [inline components/avatar/avatar.css:2]: sidebar related styles can be separated into different file
In avatar.css just keep styles related to avatar only

**Review by zoef-yt — APPROVED** [review]
Looks good. Good to merge.

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+}
+.avatar-icon{
+    margin-top: 2rem;
+    margin-bottom: 2rem;
+    padding-top: 2rem ;
+    display: flex;
+    justify-content: space-around;
+    height: 12rem;
+    width: 80%;
+    border-radius: 2px solid var(--alert-secondary-color);
```
Comment (DivyanshSareen) [inline components/avatar/avatar.css]: please check the syntax here, it should be border: 2px solid var(--alert-secondary-color);

**Review by purohitdheeraj — APPROVED** [review]
looks good for merge 👍 
just keep component styles separated

File: components/avatar/avatar.css (hunk 2 above)
Comment (Shubhamkr2610) [reply components/avatar/avatar.css]: Fixed it. PLease review

**Review by DivyanshSareen — APPROVED** [review]

**Pull request merged**